custom/icds_reports/utils/aggregation_helpers/aww_incentive.py

```diff
@@ -49,7 +58,17 @@ def aggregation_query(self):
                    awcm.block_name, awcm.supervisor_name, awcm.awc_name, awcm.aww_name,
                    awcm.contact_phone_number, awcm.wer_weighed, awcm.wer_eligible,
                    awcm.awc_days_open
-        )
+        );
+        /* update expected visits for cf cases (not in agg_ccs_record */
+        UPDATE {tablename}
+        SET expected_visits = expected_visits + ucr.expected
+        FROM (
+             SELECT SUM(0.39) AS expected, awc_id 
+             FROM {ccs_record_case_ucr} 
+             WHERE add > 183 AND closed = 0
```

Comment:
should this be `now() - add > 183` ?

Reply:
i think it wants to be `%(month)s` but yeah, it definitely should be a subtraction, will have to check the exact filter on the monthly ucr